cmem_plugin_base/dataintegration/ports.py

```diff
@@ -0,0 +1,45 @@
+"""Workflow operator input and output ports."""
+
+from typing import Sequence
+from cmem_plugin_base.dataintegration.entity import EntitySchema
+
+
+class Port:
```

Comment:
@robertisele How can we specify a Port which does not accept data ... is used just for execution order?

Reply:
ok, remember - we will do a separate dependency port ...